src/NHSD.BuyingCatalogue.Identity.Api/ViewModels/Consent/ConsentViewModel.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace NHSD.BuyingCatalogue.Identity.Api.ViewModels.Consent
+{
+    public sealed class ConsentViewModel
+    {
+        // TODO: confirm correct error message
```

Comment:
Would you mind adding the task associated with this TODO message?

Reply:
Already done: task 6329 (discussed this morning).

Reply:
Sorry I meant actually adding the task number to the comment in the code. I thought that was what we agreed in the coding standards meeting.

Reply:
Sorry, you are of course correct – I should have remembered.